libbpf-tools/fsslower.c

```diff
@@ -0,0 +1,463 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+
+/*
+ * fsslower  Trace file system operations slower than a threshold.
+ *
+ * Copyright (c) 2021 Hengqi Chen
+ *
+ * Based on fsdist and previous work by
+ *     - Brendan Gregg
+ *     - Dina Goldshtein
+ *     - Joe Yin
+ *     - Samuel Nair
+ *     - Wenbo Zhang
+ * 27-May-2021   Hengqi Chen   Created this.
```

Comment:
This is very much based on xfsslower, so I think it's fair to keep "Copyright (c) 2020 Wenbo Zhang" and "9-Mar-2020   Wenbo Zhang   Created this."?